### DISCUSSION TITLE
Why in filament 3 app content of all pages is empty?

### DISCUSSION THREAD

**Discussion by sergeynilov** [filamentphp/filament#14252] [Help]
### Package

Other

### Package Version

^3.2-stable

### How can we help you?

Something happened wit my  laravel 11 / filament 3.2 app
that content of all pages is empty :

[![enter image description here][1]][1]
Not shure what have touched to such effect.

My app.blade.php:

    <!DOCTYPE html>
    <html lang="{{ str_replace('_', '-', app()->getLocale()) }}">
        <head>
            <meta charset="utf-8">
            <meta name="viewport" content="width=device-width, initial-scale=1">
            <meta name="csrf-token" content="{{ csrf_token() }}">
    
            <title>{{ config('app.name', 'Laravel') }}</title>
    
            
            <link rel="stylesheet" href="https://fonts.bunny.net/css2?family=Nunito:wght@400;600;700&display=swap">
    
            
            @vite(['resources/css/app.css', 'resources/js/app.js'])
            @livewireStyles
        </head>
        <body class="font-sans antialiased">
            <div class="min-h-screen bg-gray-100">
                @include('layouts.navigation')
    
                
                @if (isset($header))
                    <header class="bg-white shadow">
                        <div class="max-w-7xl mx-auto py-6 px-4 sm:px-6 lg:px-8">
                            {{ $header }}
                        </div>
                    </header>
                @endif
    
                
                <main>
                    {{ $slot }}
                </main>
            </div>
        </body>
        @livewireScripts
    </html>

config/livewire.php(I did not change it) :

    <?php
    
    return [
    
        /*
        |---------------------------------------------------------------------------
        | Class Namespace
        |---------------------------------------------------------------------------
        |
        | This value sets the root class namespace for Livewire component classes in
        | your application. This value will change where component auto-discovery
        | finds components. It's also referenced by the file creation commands.
        |
        */
    
        'class_namespace' => 'App\\Livewire',
    
        /*
        |---------------------------------------------------------------------------
        | View Path
        |---------------------------------------------------------------------------
        |
        | This value is used to specify where Livewire component Blade templates are
        | stored when running file creation commands like `artisan make:livewire`.
        | It is also used if you choose to omit a component's render() method.
        |
        */
    
        'view_path' => resource_path('views/livewire'),
    
        /*
        |---------------------------------------------------------------------------
        | Layout
        |---------------------------------------------------------------------------
        | The view that will be used as the layout when rendering a single component
        | as an entire page via `Route::get('/post/create', CreatePost::class);`.
        | In this case, the view returned by CreatePost will render into $slot.
        |
        */
    
        'layout' => 'components.layouts.app',
    
        /*
        |---------------------------------------------------------------------------
        | Lazy Loading Placeholder
        |---------------------------------------------------------------------------
        | Livewire allows you to lazy load components that would otherwise slow down
        | the initial page load. Every component can have a custom placeholder or
        | you can define the default placeholder view for all components below.
        |
        */
    
        'lazy_placeholder' => null,
    
        /*
        |---------------------------------------------------------------------------
        | Temporary File Uploads
        |---------------------------------------------------------------------------
        |
        | Livewire handles file uploads by storing uploads in a temporary directory
        | before the file is stored permanently. All file uploads are directed to
        | a global endpoint for temporary storage. You may configure this below:
        |
        */
    
        'temporary_file_upload' => [
            'disk' => null,        // Example: 'local', 's3'              | Default: 'default'
            'rules' => null,       // Example: ['file', 'mimes:png,jpg']  | Default: ['required', 'file', 'max:12288'] (12MB)
            'directory' => null,   // Example: 'tmp'                      | Default: 'livewire-tmp'
            'middleware' => null,  // Example: 'throttle:5,1'             | Default: 'throttle:60,1'
            'preview_mimes' => [   // Supported file types for temporary pre-signed file URLs...
                'png', 'gif', 'bmp', 'svg', 'wav', 'mp4',
                'mov', 'avi', 'wmv', 'mp3', 'm4a',
                'jpg', 'jpeg', 'mpga', 'webp', 'wma',
            ],
            'max_upload_time' => 5, // Max duration (in minutes) before an upload is invalidated...
            'cleanup' => true, // Should cleanup temporary uploads older than 24 hrs...
        ],
    
        /*
        |---------------------------------------------------------------------------
        | Render On Redirect
        |---------------------------------------------------------------------------
        |
        | This value determines if Livewire will run a component's `render()` method
        | after a redirect has been triggered using something like `redirect(...)`
        | Setting this to true will render the view once more before redirecting
        |
        */
    
        'render_on_redirect' => true,
    
        /*
        |---------------------------------------------------------------------------
        | Eloquent Model Binding
        |---------------------------------------------------------------------------
        |
        | Previous versions of Livewire supported binding directly to eloquent model
        | properties using wire:model by default. However, this behavior has been
        | deemed too "magical" and has therefore been put under a feature flag.
        |
        */
    
        'legacy_model_binding' => false,
    
        /*
        |---------------------------------------------------------------------------
        | Auto-inject Frontend Assets
        |---------------------------------------------------------------------------
        |
        | By default, Livewire automatically injects its JavaScript and CSS into the
        | <head> and <body> of pages containing Livewire components. By disabling
        | this behavior, you need to use @livewireStyles and @livewireScripts.
        |
        */
    
        'inject_assets' => false,
    
        /*
        |---------------------------------------------------------------------------
        | Navigate (SPA mode)
        |---------------------------------------------------------------------------
        |
        | By adding `wire:navigate` to links in your Livewire application, Livewire
        | will prevent the default link handling and instead request those pages
        | via AJAX, creating an SPA-like effect. Configure this behavior here.
        |
        */
    
        'navigate' => [
            'show_progress_bar' => true,
            'progress_bar_color' => '#2299dd',
        ],
    
        /*
        |---------------------------------------------------------------------------
        | HTML Morph Markers
        |---------------------------------------------------------------------------
        |
        | Livewire intelligently "morphs" existing HTML into the newly rendered HTML
        | after each update. To make this process more reliable, Livewire injects
        | "markers" into the rendered Blade surrounding @if, @class & @foreach.
        |
        */
    
        'inject_morph_markers' => true,
    
        /*
        |---------------------------------------------------------------------------
        | Pagination Theme
        |---------------------------------------------------------------------------
        |
        | When enabling Livewire's pagination feature by using the `WithPagination`
        | trait, Livewire will use Tailwind templates to render pagination views
        | on the page. If you want Bootstrap CSS, you can specify: "bootstrap"
        |
        */
    
        'pagination_theme' => 'tailwind',
    ];
    
    vite.config.js :
    
    import { defineConfig } from 'vite';
    import laravel from 'laravel-vite-plugin';
    
    export default defineConfig({
        plugins: [
            laravel({
                input: ['resources/css/app.css', 'resources/js/app.js'],
                refresh: true,
            }),
        ],
    });

2 panels app/Providers/Filament/AdminPanelProvider.php :

    <?php
    
    namespace App\Providers\Filament;
    
    use App\Library\GetAuthPermissionHint;
    use Filament\Enums\ThemeMode;
    use Filament\Http\Middleware\Authenticate;
    use Filament\Http\Middleware\DisableBladeIconComponents;
    use Filament\Http\Middleware\DispatchServingFilamentEvent;
    use Filament\Navigation\MenuItem;
    use Filament\Panel;
    use Filament\PanelProvider;
    use Filament\Support\Colors\Color;
    use Filament\View\PanelsRenderHook;
    use Filament\Widgets;
    use Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse;
    use Illuminate\Cookie\Middleware\EncryptCookies;
    use Illuminate\Foundation\Http\Middleware\VerifyCsrfToken;
    use Illuminate\Routing\Middleware\SubstituteBindings;
    use Illuminate\Session\Middleware\AuthenticateSession;
    use Illuminate\Session\Middleware\StartSession;
    use Illuminate\View\Middleware\ShareErrorsFromSession;
    use Jeffgreco13\FilamentBreezy\BreezyCore;
    use Joaopaulolndev\FilamentEditProfile\FilamentEditProfilePlugin;
    use Joaopaulolndev\FilamentEditProfile\Pages\EditProfilePage;
    use Kenepa\ResourceLock\ResourceLockPlugin;
    use Leandrocfe\FilamentApexCharts\FilamentApexChartsPlugin;
    use Pboivin\FilamentPeek\FilamentPeekPlugin;
    use pxlrbt\FilamentEnvironmentIndicator\EnvironmentIndicatorPlugin;
    
    class AdminPanelProvider extends PanelProvider
    {
        public function panel(Panel $panel): Panel
        {
            return $panel
                ->brandName(config('app.name') . ' Admin')
                ->brandLogo(asset('images/quiz_admin_logo.jpeg'))
                ->authGuard('web')
                ->id('admin')
                ->path('admin')
                ->defaultThemeMode(ThemeMode::Dark)
    
                ->plugins([
                    FilamentPeekPlugin::make(),
                    ResourceLockPlugin::make(),
                    BreezyCore::make()
                        ->myProfile(
                            shouldRegisterUserMenu: true, // Sets the 'account' link in the panel User Menu (default = true)
                            shouldRegisterNavigation: true,
                            // Adds a main navigation item for the My Profile page (default = false)
                            hasAvatars: true, // Enables the avatar upload form component (default = false)
                            slug: 'my-profile' // Sets the slug for the profile page (default = 'my-profile')
                        ),
    
                    EnvironmentIndicatorPlugin::make()
                        ->visible(true)
                        ->color(fn() => match (app()->environment()) {
                            'production' => Color::Red,
                            'staging' => Color::Orange,
                            default => Color::Yellow,
                        })
                        ->showBadge(true)
                        ->showBorder(true),
                    FilamentApexChartsPlugin::make(),
    
                    FilamentEditProfilePlugin::make()
                        ->slug('my-profile')
                        ->setTitle('My Profile')
                        ->setNavigationLabel('My Profile')
                        ->setNavigationGroup('Group Profile')
                        ->setIcon('heroicon-o-user')
                        ->setSort(10)
                        ->canAccess(fn () => auth()->user()->id === 1)
                        ->shouldRegisterNavigation(false)
                        ->shouldShowDeleteAccountForm(true)
    //                    ->shouldShowSanctumTokens()
                        ->shouldShowBrowserSessionsForm()
                        ->shouldShowAvatarForm()
                        ->customProfileComponents([
                            \App\Livewire\CustomProfileComponent::class,
                        ]),
    
                ]) // ->plugins([ BLOCK END
    
    
                ->favicon((new \App\Library\QuizImages())->quizAdmin)
                ->navigationGroups(['Branch', 'Requests/Meetings'])
    
                ->discoverResources(in: app_path('Filament/Resources'), for: 'App\\Filament\\Resources')
                ->discoverPages(in: app_path('Filament/Pages'), for: 'App\\Filament\\Pages')
                ->pages([
                    \App\Filament\Pages\AdminDashboard::class,
                ])
                ->discoverWidgets(in: app_path('Filament/Widgets'), for: 'App\\Filament\\Widgets')
                ->widgets([
                    Widgets\AccountWidget::class,
                    Widgets\FilamentInfoWidget::class,
                ])
                ->font('DejaVu Sans')
                ->userMenuItems([
                    MenuItem::make()
                        ->label('Go to Managers area')
                        ->icon('heroicon-o-cog-6-tooth')
                        ->url('/managers'),
                    'profile' => MenuItem::make()
                        ->label(fn() => auth()->user()->name)
                        ->url(fn(): string => EditProfilePage::getUrl())
                        ->icon(getUserProfileIcon())
                ])
                ->middleware([
                    EncryptCookies::class,
                    AddQueuedCookiesToResponse::class,
                    StartSession::class,
                    AuthenticateSession::class,
                    ShareErrorsFromSession::class,
                    VerifyCsrfToken::class,
                    SubstituteBindings::class,
                    DisableBladeIconComponents::class,
                    DispatchServingFilamentEvent::class,
                ])
                ->authMiddleware([
                    Authenticate::class,
                ])
                ->renderHook(PanelsRenderHook::CONTENT_START, fn () => ' <div style="margin-top: 40px !important; margin-bottom: 10px !important;"> ' .
                                                                       ( new GetAuthPermissionHint())->get() . ' </div> ');
        }
    }

and app/Providers/Filament/ManagersPanelProvider.php :

    <?php
    
    namespace App\Providers\Filament;
    
    use App\Filament\Managers\Resources\Pages\Tenancy\EditBranchProfile;
    use App\Filament\Managers\Resources\Pages\Tenancy\RegisterBranch;
    use App\Library\GetAuthPermissionHint;
    use Filament\View\PanelsRenderHook;
    use App\Models\Branch;
    use Filament\Http\Middleware\Authenticate;
    use Filament\Http\Middleware\DisableBladeIconComponents;
    use Filament\Http\Middleware\DispatchServingFilamentEvent;
    use Filament\Navigation\MenuItem;
    
    use Filament\Pages\Auth\Register;
    use Filament\Panel;
    use Filament\PanelProvider;
    use Filament\Support\Colors\Color;
    use Filament\Widgets;
    use Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse;
    use Illuminate\Cookie\Middleware\EncryptCookies;
    use Illuminate\Foundation\Http\Middleware\VerifyCsrfToken;
    use Illuminate\Routing\Middleware\SubstituteBindings;
    use Illuminate\Session\Middleware\AuthenticateSession;
    use Illuminate\Session\Middleware\StartSession;
    use Illuminate\View\Middleware\ShareErrorsFromSession;
    use Jeffgreco13\FilamentBreezy\BreezyCore;
    use Joaopaulolndev\FilamentEditProfile\FilamentEditProfilePlugin;
    
    class ManagersPanelProvider extends PanelProvider
    {
        public function panel(Panel $panel): Panel
        {
            return $panel
                ->id('managers')
                ->path('managers')
                ->default()
                ->login()
                ->registration(Register::class)
                ->colors([
                    'danger' => Color::Rose,
                    'gray' => Color::Blue,
                    'info' => Color::hex('#55ff7f'),
                    'primary' => Color::Cyan,
                    'success' => Color::Cyan,
                    'warning' => Color::Orange,
                ])
                ->userMenuItems([
                    MenuItem::make()
                        ->label('Go To Admin')
                        ->icon('heroicon-o-cog-6-tooth')
                        ->url('/admin')
                        ->visible(fn(): bool => auth()->user() ? auth()->user()->isAdmin() : false),
    
                ])
                ->plugins([
    
                    BreezyCore::make()
                        ->myProfile(
                            shouldRegisterUserMenu: true, // Sets the 'account' link in the panel User Menu (default = true)
                            shouldRegisterNavigation: true,
                            // Adds a main navigation item for the My Profile page (default = false)
                            hasAvatars: true, // Enables the avatar upload form component (default = false)
                            slug: 'my-profile' // Sets the slug for the profile page (default = 'my-profile')
                        )
                        ->enableTwoFactorAuthentication(
                            force: false, // force the user to enable 2FA before they can use the application (default = false)
    //                        action: CustomTwoFactorPage::class // optionally, use a custom 2FA page
                        ),
    
                    /*                EnvironmentIndicatorPlugin::make()
                                        ->visible(true)
                                        ->color(fn() => match (app()->environment()) {
                                            'production' => Color::Red,
                                            'staging' => Color::Orange,
                                            default => Color::Yellow,
                                        }),*/
                    FilamentEditProfilePlugin::make()
                        ->slug('my-profile')
                        ->setTitle('My Profile')
                        ->setNavigationLabel('My Profile')
                        ->setNavigationGroup('Group Profile')
                        ->setIcon('heroicon-o-user')
                        ->setSort(10)
                        ->canAccess(fn() => auth()->user()->id === 1)
                        ->shouldRegisterNavigation(false)
                        ->shouldShowDeleteAccountForm(true)
                        ->shouldShowBrowserSessionsForm()
                        ->shouldShowAvatarForm()
                        ->customProfileComponents([
                            \App\Livewire\CustomProfileComponent::class,
                        ]),
                ])
                ->favicon((new \App\Library\QuizImages())->quiz)
                ->navigationGroups(['Our Company', 'Quizzes', 'Requests/Meetings'])
    
                ->discoverResources(in: app_path('Filament/Managers/Resources'), for: 'App\\Filament\\Managers\\Resources')
                ->discoverPages(in: app_path('Filament/Managers/Pages'), for: 'App\\Filament\\Managers\\Pages')
                ->pages([
                    \App\Filament\Pages\ManagersDashboard::class,
                ])
                ->discoverWidgets(in: app_path('Filament/Managers/Widgets'), for: 'App\\Filament\\Managers\\Widgets')
                ->widgets([
                    Widgets\AccountWidget::class,
                    Widgets\FilamentInfoWidget::class,
                ])
                ->middleware([
                    EncryptCookies::class,
                    AddQueuedCookiesToResponse::class,
                    StartSession::class,
                    AuthenticateSession::class,
                    ShareErrorsFromSession::class,
                    VerifyCsrfToken::class,
                    SubstituteBindings::class,
                    DisableBladeIconComponents::class,
                    DispatchServingFilamentEvent::class,
                ])
                ->authMiddleware([
                    Authenticate::class,
                ])
                ->tenantMiddleware([ ])
                ->tenant(Branch::class, ownershipRelationship: 'branch', slugAttribute: 'slug')
                ->tenantRegistration(RegisterBranch::class)
                ->tenantProfile(EditBranchProfile::class)
                ->renderHook(PanelsRenderHook::CONTENT_START, fn () => ' <div style="margin-top: 40px !important; margin-bottom: 10px !important;"> ' .
                                                                       ( new GetAuthPermissionHint())->get() . ' </div> ');
        }
    }

Any ideas what isa wrong and how fixed ?

  [1]: https://i.sstatic.net/M6AhY3Op.png

**Comment by leandrocfe** [comment]
_Something happened wit my laravel 11 / filament 3.2 app..._
You should verify the changes you have made to your code

**Reply by sergeynilov** [reply]
I have checked all block I modified - no idea what could raise such proble
